src/Features/Core/Portable/StackTraceExplorer/StackTraceExplorerService.cs

```diff
@@ -86,19 +85,27 @@ private static ImmutableArray<Document> GetFileMatches(Solution solution, StackF
         RoslynDebug.AssertNotNull(lineString);
         lineNumber = int.Parse(lineString);
 
+        var documentId = solution.GetDocumentIdsWithFilePath(fileName).FirstOrDefault();
+
+        if (documentId is not null)
+        {
+            var document = solution.GetRequiredDocument(documentId);
```

Comment:
This needs to be GetRequiredTextDocument since it could be an additional document ID.